utils/cliutils/commandsflags.go

```diff
@@ -579,6 +579,8 @@ const (
 	ReleaseBundles       = "release-bundles"
 	lcReleaseBundles     = lifecyclePrefix + ReleaseBundles
 	SigningKey           = "signing-key"
+	BuildName            = "build-name"
+	BuildNumber          = "build-number"
```

Comment:
No need to define the above two consts.
You should reuse the existing `buildName` and `buildNumber` consts defined above.